Yoast/wordpress-seo
# Update makeOutboundLink usage.

Supersedes https://github.com/Yoast/wordpress-seo/pull/12262

* Update usage of `makeOutboundLink` as per the changes in https://github.com/Yoast/javascript/pull/9

## Relevant technical choices:
* removes the explicitly passed props `rel={ null }` as they're no longer necessary

## Test instructions
- not necessary: test the standalone app see https://github.com/Yoast/javascript/pull/9

Fixes https://github.com/Yoast/yoast-components/issues/828
